associacao-brasileira-de-normas-tecnicas-ufrgs.csl

```diff
@@ -143,31 +180,48 @@
   </macro>
   <macro name="title">
     <choose>
-      <if type="chapter article-newspaper legislation article-magazine article-journal bill entry-encyclopedia paper-conference" match="any">
-        <text variable="title"/>
+      <if type="chapter article-newspaper article-magazine article-journal paper-conference post-weblog" match="any">
+        <text variable="title" font-weight="normal"/>
       </if>
       <else-if type="book thesis patent report" match="any">
         <text variable="title" font-weight="bold"/>
       </else-if>
+      <else-if type="entry-encyclopedia entry-dictionary" match="any">
+        <text variable="title" text-case="uppercase"/>
+      </else-if>
       <else-if type="dataset" match="any">
         <text variable="title" suffix=". "/>
       </else-if>
-      <else>
+      <else-if type="legislation bill legal_case" match="any">
```

Comment:
group with book thesis patent report above since they're the same.

Reply:
Done. Thanks!